components/runtime/squeak5/squeak5.p5m

```diff
@@ -0,0 +1,58 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 David Stes
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# see desktop/rss-glx/rss-glx.p5m:
+<transform file -> add pkg.depend.bypass-generate libGL\.so\.1>
+<transform file -> add pkg.depend.bypass-generate vm-display-X11\.so>
+
+# for the sound plugins, we only package vm-sound-pulse
+# we do not package vm-sound-OSS and vm-sound-Sun
+# on my machine, vm-sound-Sun compiles, but it does not work
+
+file mode=0555 path=usr/bin/squeak
```

Comment:
You don't need mode=0555, for bin files it's by default

Reply:
ok

Reply:
Removed mode=0555.  Note that it is essential that the mode=0555 remains in there for /usr/lib/.../squeak   .  Basically the /usr/bin/squeak script is a wrapper script which is doing some setup and then executes the /usr/lib/ <squeak plugin directory>/squeak executable.  The wrapper script must tell the squeak executable under /usr/lib where to search for the plugins.  It has not builtin plugin directory.  By default it searches for plugins in the current working directory.